src/tool_launcher.cpp

```diff
@@ -780,6 +798,29 @@ void ToolLauncher::setupHomepage()
 
 	QWidget *homepage = new QWidget(ui->stackedWidget);
 	QVBoxLayout *layout = new QVBoxLayout(homepage);
+
+	connect(PhoneHome::getInstance(), &PhoneHome::scopyVersionChanged, this, [=] () {
+		QLabel* versionLabel = new QLabel();
+
+		if (PhoneHome::getScopyVersion().toStdString().compare("") == 0) {
```

Comment:
Maybe use `PhoneHome::getScopyVersion().isEmpty()`, if I understand correctly what this is doing, it's checking if the version `QString` is empty.

Reply:
Solved.